config/_ytt_lib/github.com/cloudfoundry/capi-k8s-release/values/_default.yml

```diff
@@ -66,14 +66,18 @@ apiServer:
       -----END RSA PRIVATE KEY-----
 app_domains: []
 blobstore:
+  provider: AWS
   access_key_id: AKIAIOSFODNN7EXAMPLE
   buildpack_directory_key: cc-buildpacks
   droplet_directory_key: cc-droplets
   endpoint: http://capi-blobstore-minio.default:9000
   package_directory_key: cc-packages
-  region: ""
+  region: ~
   resource_directory_key: cc-resources
   secret_access_key: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+  google_project: project-id
```

Comment:
should we now prefix the AWS configs/creds with `aws_`?

Reply:
probably! I was trying to be as light-touch as possible with this :)